examples/bri-3/docs/zeroKnowledge/circuits/snarkjs.md

```diff
@@ -55,9 +55,12 @@ We are going to use the Groth16 zk-SNARK protocol. To use this protocol, we need
 - Phase 1 (circuit-independent): Powers of tau ceremony. This ensures that the toxic waste generated during the trusted setup is discarded and guarantees zero-knowledge of the resulting proofs, even if all participants were compromised.
 
 - Phase 2 (circuit-specific): Compiles the circuit and generates the proving and verification keys.
+  Flags:
+  1.  Protocol = Select either groth16 or plonk
+  2.  Circuit name = Pass the circuit name
 
 The commands for these two phases have been combined into ptau.sh and circuit.sh, respectively.
 
 Phase 1: `npm run snarkjs:ptau`
 
-Phase 2: `npm run snarkjs:circuit {circuitName}`
+Phase 2: `PROTOCOL={protocol} npm run snarkjs:circuit {circuitName}`
```

Comment:
why is protocol like env var and circuitName argument? i think we should have same handling of arguments/flags

Reply:
`Protocol` as env variable allows us to dynamically choose the script file to run (as per the zk protocol). This syntax is needed if we want to use a variable in the package.json file.
`circuitName` as flag allows us to use the circuitName as a variable inside the chosen script file.